test/rubygems/test_gem_package.rb

```diff
@@ -472,28 +480,32 @@ def test_extract_tar_gz_symlink_relative_path
   end
 
   def test_extract_symlink_parent
```

Comment:
Can you revert stylistic changes? We shouldn't mix it and feature changes.

Reply:
Respectfully, no.  I will remove the above two code lines.  To not waste another set of tests, I'll do so when this is resolved.

Given what I've seen in this repo:

1. If I submitted a PR with a new test and the 1st indent within the method def was one space, would I be asked to change it?

2. I've aligned the formatting of one test in a file of 16, among over two thousand test methods in total.

3. I've seen maintainers/members discuss things they admit to never having used, want benchmarks that had so many parameters affecting them as to make them pointless, and discuss engine/platform configurations they have never used anything comparable to.  I've also seen them not respond to very specific questions.

I'm just trying to make RubyGems better for myself and also users in general (including new users).  I've been writing code since I was a teenager using an acoustic-coupler and a teletype.  I started in CompSci before changing to EE/Math.  IOW, I'm old enough to say:

Y'all need to quit badgering contributors and let them improve RubyGems

Sorry for the rant...

Reply:
Unfortunately, I will never merge this until you fixed them. It's an elementary request in pull request of GitHub.

See No.4 in http://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/. It's

Reply:
> See No.4

And No.4's first sentence is: "You may feel the urge to change the formatting of the existing code to fit 'your' style. Please abstain."

I did not do that.  I formatted a test method I was already changing to match **all the other test methods**.  My editor's ruby configuration replaces a tab with two spaces.  and the non-standard spacing made it a PITA to edit.  So I fixed it.

> Unfortunately, I will never

Then I guess we're at an impasse...

Reply:
@MSP-Greg @hsbt I opened #2350 to unblock this!

Reply:
@MSP-Greg If you rebase now everyone should be happy I think! :muscle:

Reply:
Thanks.  Conflict with the rebase, just a bit more work.  Thanks, Greg